packages/block/README.md

```diff
@@ -245,179 +240,37 @@ void main()
-  )
-  const request = createCLRequest(
-    new Uint8Array([CLRequestType.Deposit, ...depositRequestBytes]),
-  ) as CLRequest<CLRequestType.Deposit>
-  const requests = [request]
-  const requestsRoot = genRequestsRoot(requests, sha256)
-
-  const block = createBlock(
-    {
-      header: { requestsHash: requestsRoot },
-    },
-    { common },
-  )
-  console.log(`Instantiated block ${block}, requestsHash=${bytesToHex(block.header.requestsHash!)}`)
-}
-
-void main()
-```
-
-Have a look at the EIP for some guidance on how to use and fill in the various deposit request parameters.
-
-#### EIP-7002 Withdrawal Requests
+Starting with v10 this library supports requests to the consensus layer which have been introduced with [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685) (`Hardfork.Prague` or higher). See the `@ethereumjs/util` [Request](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/util#module-request) README section for an overview of current request types.
 
-[EIP-7002](https://eips.ethereum.org/EIPS/eip-7002) introduces the possibility for validators to trigger exits and partial withdrawals via the execution layer. Starting with v5.3.0 this library supports withdrawal requests and a containing block can be instantiated as follows:
-
-```ts
-// ./examples/7002Requests.ts
-
-import { createBlock, genRequestsRoot } from '@ethereumjs/block'
-import { Common, Hardfork, Mainnet } from '@ethereumjs/common'
-import {
-  type CLRequest,
-  type CLRequestType,
-  bytesToBigInt,
-  createWithdrawalRequest,
-  randomBytes,
-} from '@ethereumjs/util'
-import { sha256 } from 'ethereum-cryptography/keccak.js'
-
-const main = async () => {
-  const common = new Common({
-    chain: Mainnet,
-    hardfork: Hardfork.Prague,
-  })
-
-  const withdrawalRequestData = {
-    sourceAddress: randomBytes(20),
-    validatorPubkey: randomBytes(48),
-    amount: bytesToBigInt(randomBytes(8)),
-  }
-  const request = createWithdrawalRequest(withdrawalRequestData) as CLRequest<CLRequestType>
-  const requests = [request]
-  const requestsRoot = genRequestsRoot(requests, sha256)
-
-  const block = createBlock(
-    {
-      requests,
-      header: { requestsRoot },
-    },
-    { common },
-  )
-  console.log(
-    `Instantiated block with ${
-      block.requests?.length
-    } withdrawal request, requestTrieValid=${await block.requestsTrieIsValid()}`,
-  )
-}
-
-void main()
-```
+### Consensus Types
 
-Have a look at the EIP for some guidance on how to use and fill in the various withdrawal request parameters.
+### Proof-of-Stake
 
-#### EIP-7251 Consolidation Requests
+By default (`Hardfork.Paris` (aka: Merge) and higher) block created are created as Proof-of-Stake blocks. These blocks come with their own set of header field simplifications and associated validation rules. The difficulty is set to `0` since not relevant anymore, just to name an example. For a full list of changes see [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675).
```

Comment:
```suggestion
By default (`Hardfork.Paris` (aka: Merge) and higher) blocks are created as Proof-of-Stake blocks. These blocks come with their own set of header field simplifications and associated validation rules. The difficulty is set to `0` since not relevant anymore, just to name an example. For a full list of changes see [EIP-3675](https://eips.ethereum.org/EIPS/eip-3675).
```